### DISCUSSION TITLE
Spline function for some decorations (textures)

### DISCUSSION THREAD

**Discussion by Xeavas** [FakeFishGames/Barotrauma#4708] [Ideas and suggestions]
A spline editor of sorts (not for sub shells _yet_) for decorations that have repeating textures, like wire bundles, vents, pipes and the likes of which, allowing sub builders to draw shapes to their liking. Since its decorations, there will be no hardship is trying to implement collisions for said splines. Hex seal of pain(*).

This would make creations like this (made by working joe) easier to make.
![image](https://user-images.githubusercontent.com/72674994/104196932-5a631000-545f-11eb-9975-a574b6c0aaf3.png)

(*) refer to https://discord.com/channels/103209684680323072/427654262453960724/798198342046711838

**Comment by DreamerOfRain** [comment]
Pipes and other decorative stuff like wiring could be used as test bed for other spline editable items, such as hull in the future. So this would be great.

**Reply by SoobGorl** [reply]
love how hex disliked this lmao
